android/src/main/java/nl/fhict/happynews/android/PostManager.java

```diff
@@ -79,6 +79,49 @@ public void onCompleted(Exception e, Page result) {
             });
     }
 
+
+    /**
+     * Sends content of the first page to the FeedAdapter.
+     * @param context The Application context.
+     * @param listener Implementing the LoadListener Interface.
+     */
+    public void refresh(Context context, final LoadListener listener) {
```

Comment:
A possible solution would be creating a `loadPage(int page, int size, Context context, FutureCallback<Page> callback)` method, that is called by a `refresh(Context context, LoadListener listener)` and `load(int page, int size, Context context, LoadListener listener)` method.